lib/GlobalStylesheets.js

```diff
@@ -12,6 +12,8 @@ var stylesheetIdSeed = 0;
 
 var styles = {};
 
+var browser = typeof document !== 'undefined';
```

Comment:
I would prefer this to be called `BROWSER`, but I'll merge anyway since I let this sit so long. Feel free to send another PR. Thanks for the contribution :)